pkg/liquidity-mining/contracts/gauges/avalanche/AvalancheRootGauge.sol

```diff
@@ -13,55 +13,111 @@
 // along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
 
-import "@balancer-labs/v2-interfaces/contracts/liquidity-mining/IAvalancheBridgeLimitsProvider.sol";
-
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/ERC20.sol";
 import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/SafeERC20.sol";
-import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
 
+import "./AvalancheRootGaugeLib.sol";
 import "../StakelessGauge.sol";
 
 /**
- * @dev Initiate an outgoing bridge transaction.
+ * @dev Partial interface for LayerZero BAL proxy.
  */
-interface IMultichainV4Router {
-    function anySwapOutUnderlying(
-        address token,
-        address to,
-        uint256 amount,
-        uint256 toChainID
-    ) external;
-}
+interface ILayerZeroBALProxy {
+    struct LzCallParams {
+        address payable refundAddress;
+        address zroPaymentAddress;
```

Comment:
Looks like a typo, but apparently that's really what the parameter is called. I guess because that's their token, but still confusing.

Reply:
yes, exactly